Client/Client.Core/Client.Core.Static/Client.cpp

```diff
@@ -417,121 +422,131 @@ void Client::loop()
             break;
 
             default:
-                std::cerr << "[Client][Warning] unimplemented[" << uint32_t(message.mHeader.mMessageType) << "]\n";
+                Base::Logger::FileLogger::getInstance().log
+                (
+                    "[Client][Warning] unimplemented[" + std::to_string(uint32_t(message.mHeader.mMessageType)) + "]", 
```

Comment:
**[Client][Warning]** are superfluous. Also, the type of log level should be WARNING